android/src/main/java/vn/luongvo/kmm/survey/android/ui/common/NextCircleButton.kt

```diff
@@ -1,51 +1,40 @@
 package vn.luongvo.kmm.survey.android.ui.common
 
-import androidx.compose.foundation.Image
-import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.*
 import androidx.compose.foundation.layout.size
 import androidx.compose.foundation.shape.CircleShape
-import androidx.compose.material.Button
-import androidx.compose.material.ButtonDefaults
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
 import androidx.compose.ui.graphics.Color.Companion.White
 import androidx.compose.ui.layout.ContentScale
 import androidx.compose.ui.res.painterResource
-import androidx.compose.ui.semantics.contentDescription
-import androidx.compose.ui.semantics.semantics
 import androidx.compose.ui.tooling.preview.Preview
-import androidx.compose.ui.unit.dp
 import vn.luongvo.kmm.survey.android.R
 import vn.luongvo.kmm.survey.android.ui.theme.AppTheme.dimensions
 
 @Composable
 fun NextCircleButton(
     onClick: () -> Unit,
-    contentDescription: String = ""
+    modifier: Modifier
```

Comment:
For modifier, we should have default value for it
```suggestion
    modifier: Modifier = Modifier
```

Reply:
@suho Good point 👍

Reply:
I refactored all default values for Modifier parameter here https://github.com/luongvo/kmm-survey/pull/68/commits/4757973afe0a5af450974937799013d6df98ef81